### ISSUE TITLE
Mobile app gives grey screen after update to 0.64.1

### ISSUE THREAD

**Issue by flantascience** [RocketChat/Rocket.Chat#10763]
I just updated the server version to 0.61.1.   Now, our mobile app just gives a grey screen after logging in.
![9d5d61ec-8702-4f90-ad7d-d14547da65a8](https://user-images.githubusercontent.com/12577122/40011237-2ea2977c-5775-11e8-8486-57f152b97a63.jpg)

### Server Setup Information:

* Version of Rocket.Chat Server:  0.64.1
* Operating System:  Windows
* Deployment Method(snap/docker/tar/etc):  Heroku
* Number of Running Instances:  1
* DB Replicaset Oplog:  enabled 
* Node Version:  v8.11.1
* mongoDB Version: not sure

### Steps to Reproduce:

1. Download "PUN Messenger" app on Android
2. Login using WordPress OAuth
3. After login, will be presented with the gray screen above

### Expected behavior:

I expected Rocket.Chat to open with the home page in front of me and the nav on the left side.

### Actual behavior:

Grey screen

**Comment by kaiiiiiiiii** [comment]
Do you have the same issue with the official rocket.chat+ android app?

**Comment by flantascience** [comment]
Wow... ok... this is weird.... so trying to connect with Rocket.Chat+, I get this message immediately after entering my Server Hosname: "There is a connection error".   It never even gets to the login page.

Whereas on my mobile app (which is using the Cordova rocket.chat), I get to the login screen.  The error comes after logging in.

**Comment by flantascience** [comment]
This is the server URL: https://punmessenger.peerunschooling.net

**Comment by kaiiiiiiiii** [comment]
A couple of things

1. Do you use ssl? The Rocket.Chat+ app only works with https
2. The official RC Cordova app is deprecated since the end of march (__This should also apply to any fork__).
3. Desktop Client and Browser versions working fine?


Thanks

**Comment by flantascience** [comment]
One last thing... connects fine on my iOS device.

**Comment by flantascience** [comment]
1.  Yes, I've got an SSL activated.
2.  Does this mean we're required to update to the Native app?  We can't use the Legacy at all?
3.  Desktop/Browser has no issue.

Looks like its currently isolate to Android.

**Comment by kaiiiiiiiii** [comment]
There’s still the option to compile and publish the native apps with a branding by yourself, just like you seem to do it with the cordova app.

Refs. https://github.com/RocketChat/Rocket.Chat.Cordova/blob/develop/README.md

**Comment by flantascience** [comment]
I hired someone to deploy the Cordova apps... I tried to save money by going with the Cordova ones.  The native apps didn't quite seem compatible with my OAuth login anyways.  It's been a disaster really.

Apple rejected it from their store because it wasn't using enough native features.  Android had this issue with my OAuth login not working exactly.  I opened an Issue ([10725](https://github.com/RocketChat/Rocket.Chat/issues/10725)) and was told to update the server to 0.62.1 to correct that problem... which is when this grey screen of death came in.  

So basically I've hired someone to deploy two apps... they completed their work... and now I've got zero apps.  I'm a bit down about all this, as are the kids who use the PUN website.

**Comment by flantascience** [comment]
Adn even with the native apps... my current Server isn't connecting to Rocket.Chat+ on Android, so that prolly won't help solve this problem.

**Comment by flantascience** [comment]
@kaiiiiiiiii    any idea why when trying to connect with Rocket.Chat+ on Android, I get this message immediately after entering my Server Hostname: "There is a connection error". 

It never even gets to the login page.  No issue with the Legacy Cordova app on Android, just Rocket.Chat+

**Comment by kaiiiiiiiii** [comment]
Can’t really help you with that as I don’t have an Android device to test with ^^’

I think you’ll get proper support from one of the android developers when you open an issue in the Rocket.Chat Android repository (https://github.com/RocketChat/Rocket.Chat.Android)

**Comment by flantascience** [comment]
thanks, just posted over there

Jim Flannery

jim-flannery.com
p. 860-578-4966
e. jim@jim-flannery.com
s. jim_flannery

**Comment by geekgonecrazy** [comment]
Cordova apps no longer supported.  Closing
